[2022-05-15 21:35:26,859] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-14T16:00:47.250021+00:00 [queued]>
[2022-05-15 21:35:26,875] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-14T16:00:47.250021+00:00 [queued]>
[2022-05-15 21:35:26,876] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-15 21:35:26,876] {taskinstance.py:1242} INFO - Starting attempt 2 of 2
[2022-05-15 21:35:26,877] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-15 21:35:26,892] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_run.model.jaffle_shop.int_orders> on 2022-05-14 16:00:47.250021+00:00
[2022-05-15 21:35:26,899] {standard_task_runner.py:52} INFO - Started process 200 to run task
[2022-05-15 21:35:26,903] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'model_level_dbt_dag', 'dbt_run.model.jaffle_shop.int_orders', 'scheduled__2022-05-14T16:00:47.250021+00:00', '--job-id', '207', '--raw', '--subdir', 'DAGS_FOLDER/model_level_dbt_dag.py', '--cfg-path', '/tmp/tmptiiciwvp', '--error-file', '/tmp/tmpijgnog9w']
[2022-05-15 21:35:26,904] {standard_task_runner.py:77} INFO - Job 207: Subtask dbt_run.model.jaffle_shop.int_orders
[2022-05-15 21:35:26,961] {logging_mixin.py:109} INFO - Running <TaskInstance: model_level_dbt_dag.dbt_run.model.jaffle_shop.int_orders scheduled__2022-05-14T16:00:47.250021+00:00 [running]> on host d197d9cd39af
[2022-05-15 21:35:27,015] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=model_level_dbt_dag
AIRFLOW_CTX_TASK_ID=dbt_run.model.jaffle_shop.int_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-05-14T16:00:47.250021+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-14T16:00:47.250021+00:00
[2022-05-15 21:35:27,016] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-15 21:35:27,018] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt --no-write-json run --target dev --models int_orders --profiles-dir /opt/***/dags/jaffle_shop --project-dir /opt/***/dags/jaffle_shop']
[2022-05-15 21:35:27,030] {subprocess.py:85} INFO - Output:
[2022-05-15 21:35:30,028] {subprocess.py:89} INFO - 21:35:30  Running with dbt=1.1.0
[2022-05-15 21:35:30,125] {subprocess.py:89} INFO - 21:35:30  Found 5 models, 13 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-15 21:35:30,129] {subprocess.py:89} INFO - 21:35:30
[2022-05-15 21:35:30,238] {subprocess.py:89} INFO - 21:35:30  Concurrency: 1 threads (target='dev')
[2022-05-15 21:35:30,239] {subprocess.py:89} INFO - 21:35:30
[2022-05-15 21:35:30,248] {subprocess.py:89} INFO - 21:35:30  1 of 1 START view model dev.int_orders ......................................... [RUN]
[2022-05-15 21:35:30,435] {subprocess.py:89} INFO - 21:35:30  1 of 1 OK created view model dev.int_orders .................................... [[32mCREATE VIEW[0m in 0.18s]
[2022-05-15 21:35:30,478] {subprocess.py:89} INFO - 21:35:30
[2022-05-15 21:35:30,481] {subprocess.py:89} INFO - 21:35:30  Finished running 1 view model in 0.34s.
[2022-05-15 21:35:30,493] {subprocess.py:89} INFO - 21:35:30
[2022-05-15 21:35:30,496] {subprocess.py:89} INFO - 21:35:30  [32mCompleted successfully[0m
[2022-05-15 21:35:30,502] {subprocess.py:89} INFO - 21:35:30
[2022-05-15 21:35:30,504] {subprocess.py:89} INFO - 21:35:30  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-05-15 21:35:31,553] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-15 21:35:31,580] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=model_level_dbt_dag, task_id=dbt_run.model.jaffle_shop.int_orders, execution_date=20220514T160047, start_date=20220515T213526, end_date=20220515T213531
[2022-05-15 21:35:31,626] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-15 21:35:31,686] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
